apps/hash-api/src/graph/knowledge/system-types/linear-user-secret.ts

```diff
@@ -203,8 +208,8 @@ export const getLinearSecretValueByHashWorkspaceId: ImpureGraphFunction<
       linearOrgId: integrationEntity.properties[
         SYSTEM_TYPES.propertyType.linearOrgId.metadata.recordId.baseUrl
       ] as string,
-      userAccountId: extractOwnedByIdFromEntityId(
-        integrationEntity.metadata.recordId.entityId,
+      userAccountId: extractAccountId(
+        integrationEntity.metadata.recordId.entityId as AccountEntityId,
```

Comment:
This change should be reverted, the entity here is not a user, it's a Linear Integration entity. `extractOwnedByIdFromEntityId` is correct